sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -39,7 +39,8 @@ class MicroBatchExecution(
     trigger: Trigger,
     triggerClock: Clock,
     extraOptions: Map[String, String],
-    plan: WriteToStream)
+    plan: WriteToStream,
+    catalogAndIdent: Option[(CatalogPlugin, Identifier)] = None)
```

Comment:
shall we put this info in `WriteToStream`? It's very weird to see `catalogAndIdent` as a parameter of `MicroBatchExecution`.

Reply:
Good point. `WriteToStream` is a better place for this information.